html-css/variable-naming-principles/step4-instructions.md

```diff
@@ -28,40 +26,56 @@ Click the <strong>Copy to Editor</strong> button below to add the stylesheet in
+The local custom property should represent the 5xl variation's font size. 
 
-<strong>Note: </strong> Referencing the title component documentation (https://www.patternfly.org/v4/documentation/core/components/title), note that the largest modifier size is .pf-m-4xl and that modifier class applies to `.pf-c-title`. 
+Beginning with the component name, start writing the custom property and add it to the `.pf-c-title{}` block in the `myapp.scss` file:
 
-6) We need to create a local custom property in the title component to represent the 5xl variation's font size. Beginning with the component name, let's start writing the custom property and add it to the `.pf-c-title{}` block in the `myapp.scss` file:
+It should look like: `--pf-c-title`
 
-<strong>Hint: </strong>`--pf-c-title`
+6) <strong>Add the new modifier to the custom property name.</strong>
 
-7) Next add the new modifier: `--pf-c-title--m-5xl`
+It should look like: `--pf-c-title--m-5xl`
 
-8) And finally add the property that the custom property represents: `--pf-c-title--m-5xl--FontSize`
+7) <strong>Add the property that the custom property represents.</strong>
 
-9) Now create a declaration with the global variable you added as the value for your new custom property. There should just be one line of code inside of `.pf-c-title{}`:
+It should look like: `--pf-c-title--m-5xl--FontSize`
 
-<strong>Hint: </strong>`--pf-c-title--m-5xl--FontSize: var(--pf-global--FontSize--5xl);`
+8) <strong>Add a new declaration using the global variable added in step 3 and 4.</strong>
 
-10) Next we need to write a CSS selector for the new 5xl variation. Knowing that the modifier `.pf-m-5xl` will apply to the title component, let's use the & in SCSS nested inside of .pf-c-title to create `.pf-c-title.pf-m-5xl`.
+Add this declaration inside of `.pf-c-title{}`.
 
-<strong>Hint: </strong> Add this block inside of `.pf-c-title`:
+It should look like: `--pf-c-title--m-5xl--FontSize: var(--pf-global--FontSize--5xl);`
+
+9) <strong>Create a CSS selector for the new 5xl variation.</strong>
+
+Knowing that the modifier `.pf-m-5xl` will apply to the title component, use the & in SCSS to nest `.pf-m-5xl` inside of `.pf-c-title` to create `.pf-c-title.pf-m-5xl`.
+
+Add this block inside of `.pf-c-title`, underneath the declaration from step 8.
 
 ```
 &.pf-m-5xl {
   //Code goes here
 }
 ```
 
-11) Last all we need to do is create a declaration within the modifier to define the font-size property with a value of the new title component custom property we created for this variation. Add this line inside of `.pf-m-5xl{}`
+10) <strong>Create a declaration within the modifier block.</strong>
+
+The declaration will define the font size property with a value of the new title component custom property that was created for this variation. 
```

Comment:
sb: The declaration will define the font-size property with a value of the new title component custom property created for this variation.